problemsets-mat223/linearalbegra.tex

```diff
@@ -1719,6 +1742,11 @@ \section*{Task 1.3: The Magic Carpet, Getting Back Home}
 				vectors. 
 			\end{solution}
 		\item Is $\Set{\vec u,\vec v,\vec w,\vec r}$ linearly independent?
+			\begin{solution}
+				No. We know from earlier that $\vec u+\vec v+2\vec w=\vec 0$, and
```

Comment:
-2\vec w

Reply:
It's +, right? The original equation is:

3u + 2v + w = 2u + v - w

Bringing everything to the left side:

(3u - 2u) + (2v - v) + (w - (-w)) = 0

Reply:
My mistake!